File: codigos/multiples.c
#include <stdio.h>
#include <math.h>



int countMultiples(int a,int b, int c)
{	
	int count = 0;
	int aux;
	
    if (a%c == 0 || b%c == 0)
        count +=1;
	
    aux = (b-a);

    if (aux+c>b){
	
        count += ((b-a)/c);
        return count;
	}
	aux = (double)((b-a)/c);
    count +=ceil(aux);
    
    return count;
}

int main()
{
	int a,b,c;
	int count;
	
	scanf("%d",&a);
	scanf("%d",&b);
	scanf("%d",&c);
	
	count = countMultiples(a,b,c);
	
	printf("%d\n",count);
	
	
}

File: codigos/multiples.py
# caclcula quantos multiplos de C existe(m) entre A e B
import math as m

def countMultiples(a,b,c):
    count = 0
    if a%c == 0 or b%c == 0:
        count += 1
    aux = (b-a)
    if aux+c>=b:
        count += m.floor((aux)/c)
    else:
        count += m.ceil((aux)/c)

    return count

def main():
    a = int(input())
    b = int(input())
    c = int(input())
    count = countMultiples(a,b,c)

    print("{0}".format(int(count)))

main()
